libs/adapters/src/trpc/index.ts

```diff
@@ -212,7 +219,14 @@ export const query = <Input extends ZodSchema, Output extends ZodSchema>(
         method: 'GET',
         path: `/${Tag.Query.toLowerCase()}/${factory.name}`,
         tags: [Tag.Query],
-        headers: [{ name: 'address_id' }],
+        headers: [
```

Comment:
Are we not moving this to JWT as discussed or handling later?

Reply:
Yes, this is just to get the spec to build, it was missing the type